blog/2015/04/workshops-for-companies.html

```diff
@@ -0,0 +1,68 @@
+---
+layout: blog
+root: ../../..
+author: Matt Davis
+title: "Workshops for Companies"
+date: 2015-04-09
+time: "16:00:00"
+category: ["Workshops"]
+---
+<!-- start excerpt -->
+<p>
+  At our March 12, 2015 meeting, the Software Carpentry Foundation Steering
+  Committee discussed whether and under what terms to provide workshops to
+  for-profit corporations. There were no objections to the idea of doing
+  workshops for corporations, but it is something new to Software Carpentry.
+  We've decided to run a pilot program encompassing five corporate workshops
+  so we can learn more about working with corporations. The pilot program
+  will allow us to gauge corporate interest and collect feedback from
+  instructors, coordinators, and others about how things go.
+</p>
+<!-- end excerpt -->
+<p>
+  During the pilot, workshops will be run on these terms:
+</p>
+<ol>
+  <li>
+    Hosts will pay $5000 for their first workshop and $3000 for subsequent workshops.
+  </li>
+  <li>
+    As with our other workshops,
+    instructors will be volunteers and their travel expenses will be covered by hosts.
+  </li>
+  <li>
+   Three-quarters of the money directed to the SCF will be used
+   to underwrite administrative fees for non-profit organizations.
+  </li>
+  <li>
+    Fee waivers/discounts will be available to corporations on a
+    case-by-case basis just as with non-profit organizations.
+  </li>
+</ol>
+<p>
+  The third point is probably the most important:
+  each workshop at a company will effectively pay the administration fee
+  for three workshops at universities,
+  for under-represented groups,
+  etc.
+</p>
+<p>
+  We have updated <a href="{{page.root}}/scf/workshops.html">our workshop operations description</a>
+  to reflect this new policy.
+  At the end of the pilot program the Steering Committee will consider
+  whether to continue doing workshops for for-profit entities, taking into
+  account such factors as whether the instructors, coordinators, and students
+  involved had positive experiences, and whether there seems likely to be
+  continued interest from corporations. The Steering Committee may also elect
+  to change the terms under which corporate workshops are provided.
+  (Because it has already been arranged, the upcoming workshop at Monsanto
+  will pay the lower administrative fee of $1250, but it will count toward
+  the five-workshop sample for the pilot program.)
```

Comment:
I'd suggest explicitly covering what happens after the fifth for-a-for-profit workshop has been arranged but before the committee has decided on the new plan.  See my [discuss@ email][1] for details.

[1]: http://lists.software-carpentry.org/pipermail/discuss_lists.software-carpentry.org/2015-April/002885.html